src/Spectre.libPlatform/Math.h

```diff
@@ -351,4 +352,34 @@ constexpr std::vector<DataType> abs(gsl::span<const DataType> data)
     static_assert(std::is_arithmetic_v<DataType>, "DataType: expected arithmetic.");
     return Functional::transform(data, [](DataType entry) { return std::abs(entry); });
 }
+
+/// <summary>
+/// transform to opposite values
+/// </summary>
+/// <param name="data">The data.</param>
+/// <returns>Vector consisting of opposite values of elements of first</returns>
+template <class DataType>
+constexpr std::vector<DataType> negate(gsl::span<const DataType> data)
+{
+    static_assert(std::is_arithmetic_v<DataType>, "DataType: expected arithmetic.");
+    return Functional::transform(data, [](DataType entry) { return std::negate<DataType>(entry); });
+}
+
+/// <summary>
+/// fill with random values
+/// </summary>
+/// <param name="data">The data.</param>
+/// <returns>Vector consisting of opposite values of elements of first</returns>
+template <class DataType>
+constexpr std::vector<DataType> fillWithRandom(gsl::span<const DataType> empty_data, libGenetic::Seed seed, double probability, int size)
```

Comment:
I've meant even more general solution. The function I thought of would be called like `build`, accept `size` and `generator`. `size` would specify the number of elements, `generator` would produce consecutive elements. This way we would just pass a lambda from the outside, which draws a single bool from a distribution. Just another variation on `Transform`.